tests/system/providers/elasticsearch/example_elasticsearch_query.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for Elasticsearch Query.
+"""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.decorators import task
+from airflow.providers.elasticsearch.hooks.elasticsearch import ElasticsearchHook
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = 'elasticsearch_dag'
+CONN_ID = 'elasticsearch_default'
+
+
+@task(task_id='es_print_tables')
+def show_tables():
+    """
+    show_tables queries elasticsearch to list available tables
+    """
+    es = ElasticsearchHook(elasticsearch_conn_id=CONN_ID)
+
+    # Handle ES conn with context manager
+    with es.get_conn() as es_conn:
+        tables = es_conn.execute('SHOW TABLES')
+        for table, *_ in tables:
+            print(f"table: {table}")
+    return True
+
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval="@once",
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example", "elasticsearch"],
+) as dag:
+    # [START howto_elasticsearch_query]
+    execute_query = show_tables()
```

Comment:
Note that sphinx will only take content between START and END markers. So your documentation will show:
```
execute_query = show_tables()
```
which is not telling much. Consider moving the markers to where you define the show_tables:
```
 [START howto_elasticsearch_query]
@task(task_id='es_print_tables')
def show_tables():
(method body)
 [END howto_elasticsearch_query]
```

Reply:
Yep. Moving them to inside of the method works even better.